### ISSUE TITLE
[FR] allow option for stream sources to be persistently loaded for faster viewing.

### ISSUE THREAD

**Issue by rbray89** [AlexxIT/go2rtc#330]
I know this may not apply as much for anyone using Frigate where their RTSP streams are constantly loaded, but for view only needs (baby monitors, pantry cams, etc.) It would be nice to have an option where the streams could be pre-loaded so we don't have to wait for the ffmpeg startup delay.

**Comment by AlexxIT** [comment]
FFmpeg has delay only for video in stream. Even if it is not transcoded.
If you do not need to transcode video - this config is recommended:
```
streams:
  camera1:
  - rtsp://...
  - ffmpeg:camera1#audio=xxx
```

It starts up much faster

**Comment by rbray89** [comment]
Yeah, I'm aware. My cameras only support 4k in h265 and 640p in h264, so I'm using HW tanscoding to both downscale the 4k stream, and alter the key-frame down to .5s for a preview stream.

The ability to tweak stream settings is an amazing feature for cameras with limited configuration,  and let's me standardize streams across vendors to the same framerate, key interval, and resolution.

**Comment by rbray89** [comment]
Also, if we're transcoding for Mjpeg, would allow for more instantaneous snap-shots.

**Comment by AlexxIT** [comment]
I think there are ffmpeg settings to speed up the start. I just don't know them.
Now ffmpeg starts with a delay equal to the size of the keyframe of the camera. I think it needs to catch two keyframes before starting (by default).

**Comment by sandstormkeshav** [comment]
> Yeah, I'm aware. My cameras only support 4k in h265 and 640p in h264, so I'm using HW tanscoding to both downscale the 4k stream, and alter the key-frame down to .5s for a preview stream.
> 
> The ability to tweak stream settings is an amazing feature for cameras with limited configuration, and let's me standardize streams across vendors to the same framerate, key interval, and resolution.

@rbray89 Could you please share your config, I am trying to do something similar with downscaling a stream and setting key frame. Thank you very much!
